plugin-server/src/config/config.ts

```diff
@@ -233,6 +233,7 @@ export function getDefaultConfig(): PluginsServerConfig {
         SESSION_RECORDING_V2_S3_REGION: 'us-east-1',
         SESSION_RECORDING_V2_S3_ACCESS_KEY_ID: 'object_storage_root_user',
         SESSION_RECORDING_V2_S3_SECRET_ACCESS_KEY: 'object_storage_root_password',
+        SESSION_RECORDING_V2_S3_TIMEOUT_MS: 5000,
```

Comment:
No idea but could this be too short? Like could a valid upload take that long?

Reply:
Good point – had a look at the metrics, I'll change it – 30 seconds should be fine.